tests/test_client_session.py

```diff
@@ -918,7 +918,7 @@ def to_url(path: str) -> URL:
             assert to_trace_urls(on_request_redirect) == []
             assert to_trace_urls(on_request_end) == [to_url("/?x=0")]
             assert to_trace_urls(on_request_exception) == []
-            assert to_trace_urls(on_request_chunk_sent) == [to_url("/?x=0")]
```

Comment:
This seems like a bug that we sent this trace when there was nothing actually sent so I didn't preserve this behavior